common/helpers.go

```diff
@@ -247,3 +251,15 @@ func convert[T any](from any, to *T) error {
 	}
 	return json.Unmarshal(buf, to) //nolint:wrapcheck
 }
+
+// TODO:
```

Comment:
Shall we create an issue for this so that we don't forget?

Reply:
Thanks! Done: https://github.com/grafana/xk6-browser/issues/1192.